packages/concerto-cto/test/cto/field.cto

```diff
@@ -2,4 +2,8 @@ namespace test.field
 
 concept Foo {
   o String foo default="FOO" optional
-}
\ No newline at end of file
+  o Double doubleDefaultZero default=0.0 optional
+  o Integer intDefaultZero default=0 optional
+  o Long longDefaultZero default=0 optional
+  o Boolean boolDefaultFalse default=false optional
```

Comment:
Maybe a case for default value not even set?

Reply:
I don't think that we should add a default modifier in the JSON serialization of a model, if the author didn't specify one in the CML. The behaviour is consistent now with this PR.